Mastodon/Scene/HomeTimeline/HomeTimelineViewModel+LoadLatestState.swift

```diff
@@ -51,7 +51,7 @@ extension HomeTimelineViewModel {
 extension HomeTimelineViewModel.LoadLatestState {
     class Initial: HomeTimelineViewModel.LoadLatestState {
         override func isValidNextState(_ stateClass: AnyClass) -> Bool {
-            return stateClass == Loading.self
+            return stateClass == Loading.self || stateClass == LoadingManually.self
```

Comment:
`HomeTimelineViewModel.LoadOldestState` is not affected, as there's no `LoadManually`-State (aka pull to refresh), right?